models/user/user.go

```diff
@@ -742,7 +749,7 @@ func CreateUser(u *User, overwriteDefault ...*CreateUserOverwriteOptions) (err e
 		return err
 	}
 
-	if err = db.Insert(ctx, u); err != nil {
+	if _, err = db.GetEngine(ctx).NoAutoTime().Insert(u); err != nil {
```

Comment:
No. This will disable to generate a current time to `CreatedUnix` and `UpdatedUnix` even if they are zero. It will break the logic of other places.